app/models/settings.rb

```diff
@@ -7,8 +7,9 @@ def self.registrations_enabled?
   end
 
   def self.toggle_registrations
-    registrations_enabled = Settings.find_by(key: :REGISTRATIONS_ENABLED)
-    registrations_enabled ||= Settings.create(key: :REGISTRATIONS_ENABLED, value: "true")
+    registrations_enabled = Settings.find_or_initialize_by(key: :registrations_enabled)
+    registrations_enabled.value = "false" unless registrations_enabled.value
+    registrations_enabled.save
 
     registrations_enabled.value = registrations_enabled.value == "true" ? "false" : "true"
     registrations_enabled.save
```

Comment:
This should now be deleted, right?

Reply:
Nope.... because we store strings and the code at line 11 is to set the default value if it's `nil`.

Reply:
but yes we can only save it once

Reply:
Why are we setting twice? We could do it with a `case` statement

```ruby
registrations_enabled.value = case registrations_enabled.value
  when nil, "false"
    "true"
  when "true"
    "false"
end
```

Reply:
Used to javascript, forgot that you could that with Ruby

Reply:
I know the feeling #elixir 😞